wqflask/wqflask/correlation/pre_computes.py

```diff
@@ -6,29 +6,40 @@
 from base.data_set import query_table_timestamp
 from base.webqtlConfig import TMPDIR
 
+from json.decoder import JSONDecodeError
+
 
 def fetch_all_cached_metadata(dataset_name):
     """in a gvein dataset fetch all the traits metadata"""
     file_name = generate_filename(dataset_name, suffix="metadata")
 
-    file_path = os.path.join(TMPDIR, file_name)
+    file_path = Path(TMPDIR, file_name)
```

Comment:
**Question**: What's the preference of this over "os.path.join"? Here I'm just being curious.

Reply:
Actually, I later see that you use the "unlink" and "touch" so :+1: